certified-connectors/SigningHub/apiProperties.json

```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "capabilities": [],
+	"publisher": "Ascertia",
+    "stackOwner": "Ascertia",
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "0123456789",
```

Comment:
@jhan-zaib You can just keep it as `"[[TEST]]" or `"[[DUMMY]"`.

Reply:
added dummy text